RibonDAO/core-api
# Feat/authentication

# Description

- Authentication

# Necessary changes

- check google client id env vars

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Test donation with old user and new user and with all methods
- Test create account with all methods
- Test link expired
- Test logout
- Test extra ticket, check if it is possible get more tickets with that integration
- Test delete account
- Test whether the native app is still working
